__tests__/utils/errorHandler.test.js

```diff
@@ -36,3 +47,20 @@ describe("Test group for 'normalizeError' method", () => {
     expect(errMessage).toBe(`${staticMessage} "${JSON.stringify(obj)}"`);
   });
 });
+
+describe("Test group for 'handleError' method", () => {
```

Comment:
if we test after setTimeout lets add the done argument in test method and use it after expect

Reply:
Addressed

Reply:
Addressed

Reply:
I don't think this is addressed.

Reply:
This feels like a hack. Let's discuss this.

Reply:
Done